keyboards/pkb65/readme.md

```diff
@@ -0,0 +1,14 @@
+# PKB65

```

Comment:
Please use Unix line endings for this file.

Reply:
reuploaded the readme, please see the most recent commit

Reply:
Still has Windows line endings.

Reply:
ok check it now, should be Unix LF now sorry!